src/test/java/tests/boardTests/listTests/cardTests/CardTests.java

```diff
@@ -106,4 +106,28 @@ public void movingCardsBetweenLists() {
         softAssert.assertTrue(enableCardOnInitialList, "The card with name: " + cardName + "does not exist in this list with name " + listName);
         softAssert.assertTrue(enableCardOnTargetList, "The card with name: " + cardName + "does not exist in this list with name " + customListName);
     }
+
+    @Test(description = " 6.1. Search by the content of boards and cards")
+    @Description("PJ2024-20")
+    public void searchByContent() {
+        String boardName = boardBody.getName();
+        String cardName = cardBody.getName();
+        List<String> searchBoardResult, searchCardResult;
+
+        apiCardClient.createNewCard(cardBody,listId,HTTP_OK);
+        refresh();
+        trelloHomePage.getTrelloHomeHeaderFragment().getSearchField().sendKeys(boardName);
```

Comment:
in getSearchFiled() can you do some other activities? or just sentKeys()?
I guess you can do  sentKeys() into that method, if don't, please explain to me

Reply:
i can clean it :)